CDCgov/NEDSS-Modernization
# adding links in add patient and adding error validation for address

Enabling links missing in Add patient page and adding validation to the second address field in add patient page as well



## Tickets
https://cdc-nbs.atlassian.net/browse/CNFT1-1461
https://cdc-nbs.atlassian.net/browse/CNFT1-1610

## Steps to verify


1. Navigate to the add patient page
2. observe and use the links added in the left bar of the add patient page
3. Navigate to the second address field and test by adding more than 100 characters, step out of the field and see the error message
